dessia_common/serialization.py

```diff
@@ -160,7 +161,8 @@ def serialize_with_pointers(value, memo=None, path='#', id_method=True, id_memo=
             id_ = str(uuid.uuid1())
             id_memo[id_] = serialized
             memo[value] = f'#/_references/{id_}'
```

Comment:
I am not sure  this is enough to resolve the issue. If my understunding is correct, this will still add a new reference in _references, on backend side, we can't tell if this ref is for a standalone obj or not ( unless we do some code inspection i guess) and so it will be saved as a separate entry in db.